engine/server/sv_client.c

```diff
@@ -2091,6 +2091,744 @@ static qboolean SV_SendBuildInfo_f( sv_client_t *cl )
 	return true;
 }
 
+edict_t* pfnFindEntityInSphere(edict_t* pStartEdict, const float* org, float flRadius);
+
+static edict_t* SV_GetCrossEnt(edict_t* player)
```

Comment:
Reformat this patch following our code style: https://github.com/FWGS/xash3d-fwgs/blob/master/CONTRIBUTING.md#code-style